scripts/core/superdesk-api.d.ts

```diff
@@ -366,18 +367,21 @@ declare module 'superdesk-api' {
         ): void;
         getLabel(item: unknown): string;
         getId(item: unknown): string;
-        canSelectBranchWithChildren?(branch: ITreeNode<unknown>): boolean;
+        canSelectBranchWithChildren?: boolean;
         optionTemplate?: React.ComponentType<{item: unknown}>;
         valueTemplate?: React.ComponentType<{item: unknown}>;
         multiple: boolean;
     }
 
+    /**
+     * @deprecated Use IDropdownConfigManualSource/IDropdownConfigVocabulary/IDropdownConfigRemoteSource
```

Comment:
Not sure if we need to deprecate it. Maybe it's enough to warn not to use this one if use case can be satisfied using other dropdown sources.